packages/qwik/src/core/render/jsx/types/jsx-qwik-elements.ts

```diff
@@ -20,7 +30,9 @@ export interface QwikIntrinsicAttributes {
 /**
  * @public
  */
-export interface QwikIntrinsicElements extends IntrinsicHTMLElements {
+export interface QwikIntrinsicElements extends IntrinsicElements {
   script: QwikScriptHTMLAttributes<HTMLScriptElement>;
-  [key: string]: QwikCustomHTMLAttributes<QwikCustomHTMLElement>;
+  [key: string]:
+    | QwikCustomHTMLAttributes<QwikCustomHTMLElement>
+    | QwikCustomSVGAttributes<QwikCustomSVGElement>;
```

Comment:
This probably isn't accurate, I think doing this you'll be able to pass SVG attributes to HTML elements...which I think we don't want?

Reply:
This comment concerns me....

Also could we have some tests which dave `satisfies` to verify that TS is now happy with then new types?